### ISSUE TITLE
Broken node support

### ISSUE THREAD

**Issue by tornqvist** [yoshuawuyts/choo#388]
`app.toString('/')` does not work in node due to two recent changes in sheet-router and barracks.

The problem with sheet router is due to accessing `global.location.origin` which throws an error seeing as `global.location` is undefined. This results in the script crashing on execution when being required. See commit: https://github.com/yoshuawuyts/sheet-router/commit/c07ec03c6ba3b265ce5617f328cd7086830ee6eb

The problem with barracks is a recent addition of emitting the `init` event on start up. In choo this triggers the render function that tries to call  `window.requestAnimationFrame` (through nanoraf) which does not exist in node. See commit: https://github.com/yoshuawuyts/barracks/commit/d57d69a98d7eab0048925db59ea673a019738e84

For some reason the tests for both sheet-router and choo seem to work fine, but the following example (taken from the tests) crash in node.

```javascript
const choo = require('choo')
const html = require('choo/html')
const app = choo()
app.router(['/', () => html`<h1>Hello Tokyo!</h1>`])
app.toString('/')
```

**Comment by yoshuawuyts** [comment]
oops, nice catch - yeah we should fix this asap

**Comment by yoshuawuyts** [comment]
I think this should be solved in `v4.0.4` - lmk if this works for you (:

**Comment by tornqvist** [comment]
Works like a charm, thanks!
